src/mmw/sass/components/_charts.scss

```diff
@@ -5,7 +5,6 @@
 .chart-container {
   width: 100%;
   height: 100%;
-  margin-top: 30px;
```

Comment:
Consider switching this to `margin-bottom` to add some space between the chart and graph. It's currently a little tight:

![image](https://cloud.githubusercontent.com/assets/1042475/12830798/1fe2e696-cb5f-11e5-884a-07ab6235c2ab.png)